catt/cli.py

```diff
@@ -73,19 +73,19 @@ def process_path(ctx, param, value):
     return path
 
 
-def get_device(ctx, param, value):
-    try:
-        return ctx.default_map["aliases"][value]
-    except KeyError:
+def process_device(ctx, param, value):
```

Comment:
This might need a docstring and some better naming of variables, since `value` doesn't tell me what the value should be.

Reply:
I like all the callback functions being similar in this regard. But let me add the docstring.

Reply:
Oh, with `value`? That's fine, as long as the docstring explains what the value can be.